### ISSUE TITLE
error: non-constant-expression cannot be narrowed from type 'double' to 'float' in initializer list

### ISSUE THREAD

**Issue by ryandesign** [thezbyg/gpick#158]
When I try to build the latest gpick code, using the scons build system and adding `-std=c++14`, I get 18 errors of this type:

```
/usr/bin/clang++ -o build/source/transformation/Quantization.o -c -Os -std=c++14 -stdlib=libc++ -arch x86_64 -arch i386 -Os -arch x86_64 -arch i386 -D_REENTRANT -DENABLE_NLS -DGSEAL_ENABLE -Iextern -I/opt/local/include/gtk-2.0 -I/opt/local/lib/gtk-2.0/include -I/opt/local/include/pango-1.0 -I/opt/local/include/gio-unix-2.0 -I/opt/local/include/cairo -I/opt/local/include/atk-1.0 -I/opt/local/include/pixman-1 -I/opt/local/include/gdk-pixbuf-2.0 -I/opt/local/include/libpng16 -I/opt/local/include/harfbuzz -I/opt/local/include/fribidi -I/opt/local/include/glib-2.0 -I/opt/local/lib/glib-2.0/include -I/opt/local/include/ossp -I/opt/local/include/freetype2 -Isource source/transformation/Quantization.cpp
source/transformation/ColorVisionDeficiency.cpp:122:3: error: non-constant-expression cannot be narrowed from type 'double' to 'float' in initializer list [-Wc++11-narrowing]
                rgb_anchor[1] * anchor[8] - rgb_anchor[2] * anchor[7],
                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
source/transformation/ColorVisionDeficiency.cpp:122:3: note: insert an explicit cast to silence this issue
                rgb_anchor[1] * anchor[8] - rgb_anchor[2] * anchor[7],
                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
                static_cast<float>(                                  )
```

**Comment by thezbyg** [comment]
Should be fixed in commit ef5aff9af70ce17806b23e0c519ebf122a88c7b6.

**Comment by ryandesign** [comment]
Thank you! Yes, that fixes it.
